### PR TITLE
Try forcing manual mounting on setup

### PR DESCRIPTION
**Pull request by ImUrX**
Many people that start using the app end up using automatic without understanding how to use it. So let's enforce the manual mounting on the first-time use

### CODE REVIEW

**Review by ButterscotchV — APPROVED** [review]

**Comment by ImUrX** [comment]
I changed on how this works, and now we are using a modal for it as a warning sign

**Review by loucass003 — APPROVED** [review]

**Review by ButterscotchV — APPROVED** [review]
Looks fine, we should probably also do this for AutoBone, especially since you don't really even do a full reset before getting to that step in the setup wizard

File: gui/public/i18n/en/translation.ftl [hunk 1]
```diff
@@ -720,7 +720,12 @@ onboarding-choose_mounting-manual_mounting = Manual mounting
 # Italized text
 onboarding-choose_mounting-manual_mounting-label = Recommended
 onboarding-choose_mounting-manual_mounting-description = This will let you choose the mounting direction manually for each tracker
-
+# Multiline text
+onboarding-choose_mounting-manual_modal-title = Are you sure you want to do
+    automatic mounting?
```
Comment (Louka3000) [inline gui/public/i18n/en/translation.ftl:725]:
```suggestion
onboarding-choose_mounting-manual_modal-title = Are you sure you want to do
    the automatic mounting calibration?
```

File: gui/public/i18n/en/translation.ftl [hunk 2]
```diff
@@ -720,7 +720,12 @@ onboarding-choose_mounting-manual_mounting = Manual mounting
...
+onboarding-choose_mounting-manual_modal-description = <b>You should first try doing manual mounting</b>, automatic mounting is quite hard to get right in its poses and requires some practice.
```
Comment (Louka3000) [inline gui/public/i18n/en/translation.ftl:726]:
```suggestion
onboarding-choose_mounting-manual_modal-description = <b>We recommend the manual mounting calibration for new users</b>, as the automatic mounting calibration's pauses can be hard to get right first and may require some practice.
```

**Review by Louka3000 — CHANGES_REQUESTED** [review]

File: gui/public/i18n/en/translation.ftl (hunk 2 above)
Comment (ImUrX) [reply gui/public/i18n/en/translation.ftl:726]:
```suggestion
onboarding-choose_mounting-manual_modal-description = <b>We recommend the manual mounting calibration for new users</b>, as the automatic mounting calibration's poses can be hard to get right first and may require some practice.
```

File: gui/public/i18n/en/translation.ftl [hunk 3]
```diff
@@ -720,7 +720,12 @@ onboarding-choose_mounting-manual_mounting = Manual mounting
 # Italized text
 onboarding-choose_mounting-manual_mounting-label = Recommended
 onboarding-choose_mounting-manual_mounting-description = This will let you choose the mounting direction manually for each tracker
-
+# Multiline text
+onboarding-choose_mounting-manual_modal-title = Are you sure you want to do
+    the automatic mounting calibration?
+onboarding-choose_mounting-manual_modal-description = <b>We recommend the manual mounting calibration for new users</b>, as the automatic mounting calibration's poses can be hard to get right first and may require some practice.
```
Comment (Louka3000) [inline gui/public/i18n/en/translation.ftl:726]:
```suggestion
onboarding-choose_mounting-manual_modal-description = <b>The manual mounting calibration is recommended for new users</b>, as the automatic mounting calibration's poses can be hard to get right first and may require some practice.
```

**Review by Louka3000 — APPROVED** [review]
Lgtm, but didn’t check the code because it’s typescript

**Pull request merged**